buildSrc/src/main/java/Dependencies.kt

```diff
@@ -3,8 +3,8 @@ import org.gradle.util.VersionNumber
 
 object BuildPluginsVersion {
     val AGP = System.getenv("VERSION_AGP") ?: "7.4.0"
-    const val DOKKA = "1.6.10"
-    const val KOTLIN = "1.6.10"
+    const val DOKKA = "1.8.10"
+    const val KOTLIN = "1.8.20"
```

Comment:
just for me to know: is the bump needed to use new compiler plugin APIs? Or compose? or both? I think it's fine to bump it, if we clean up `plugin-build/build.gradle.kts`, otherwise we'd be leaking our version into consumer's classpath which is probably not great.

Reply:
It's required for the Compose Compiler Plugins, as there was a breaking change in the APIs for `irTemporary()`